packages/plugins/Gfi/src/components/List.vue

```diff
@@ -0,0 +1,159 @@
+<!-- eslint-disable vue/no-v-html -->
+<template>
+  <v-scroll-x-reverse-transition>
+    <v-card class="plugin-gfi-list">
```

Comment:
```suggestion
    <v-card>
```
If all three other suggestions are accepted, the class is no longer needed.

Reply:
One is not.